app/src/main/java/org/mozilla/webmaker/activity/Play.java

```diff
@@ -15,10 +19,23 @@ public Play() {
         super("project", R.id.page_layout, R.layout.page_layout, R.menu.menu_play);
     }
 
+    protected String login_username;
+    protected String app_creator;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
+        try {
+            SharedPreferences userSession = getSharedPreferences(WebAppInterface.USER_SESSION_KEY, 0);
+            JSONObject session = new JSONObject(userSession.getString("session", ""));
+
+            login_username = session.getJSONObject("user").get("id").toString();
+            app_creator = routeParams.get("user").toString();
+        } catch (JSONException e) {
+            e.printStackTrace();
```

Comment:
You can `// do nothing` here. No need to print the stack trace is a user session does not exist (which will be common).